WHIR/Views/sv.lproj/About.strings

```diff
@@ -0,0 +1,6 @@
+
+/* Class = "UINavigationItem"; title = "About"; ObjectID = "RwD-W3-TVP"; */
+"RwD-W3-TVP.title" = "Om";
+
+/* Class = "UITextView"; text = "Handcrafted by © 2018 Bruce Röttgers\n\nLogo by Zlatko Najdenovski on Flaticon. Licensed under the Flaticon Basic License (https://www.flaticon.com/authors/zlatko-najdenovski).\n Barcode icon made by Pixel perfect on Flaticon (https://www.flaticon.com/free-icon/barcode_726558). \nUsing CocoaPod KMPlaceholderTextView by MoZhouqi (https://github.com/MoZhouqi/KMPlaceholderTextView). Licensed under the MIT license.\n\nThanks for reading and using this App ❤.\n\nFeel free to contact me on my support email (can be found on the App Store page), if you find bugs or just to say hello. (br.apps@icloud.com) "; ObjectID = "lpw-Ku-zCI"; */
+"lpw-Ku-zCI.text" = "Handgjord av © 2018 Bruce Röttgers och flera andra på GitHub\n\nLogo av Zlatko Najdenovski på Flaticon. Licensierad enligt Flaticon Basic License (https://www.flaticon.com/authors/zlatko-najdenovski).\n Streckkod ikon gjord av Pixel perfect på Flaticon (https://www.flaticon.com/free-icon/barcode_726558). \nAnvänder CocoaPod KMPlaceholderTextView av MoZhouqi (https://github.com/MoZhouqi/KMPlaceholderTextView). Licensierad enligt MIT license.\n\nTack för att du läste och använde den här appen ❤.\n\nKänna sig fri att kontakta mig med min support e-postadress (kan hittas på den App Store sida), om du hittar buggar eller bara att säga hej. (br.apps@icloud.com) ";
```

Comment:
I translated this string into Swedish, and it's currently in `master`. What happened to it?